admin/display/general-metaboxes.php

```diff
@@ -51,11 +51,11 @@ static function display_extra_metaboxes( $add, $meta ) {
 				</div>
 				<?php
 					// Is this fall through deliberate?
-				case 'aioseop-donate':
```

Comment:
This file appears to have incorrect indents. Doesn't appear to have any changes, so this file's changes could probably just be discarded.

Reply:
I left his as is just for now.